### ISSUE TITLE
Feature: Flag a migration as tenant based.

### ISSUE THREAD

**Issue by BenMorganIO** [Dania02525/apartmentex#39]
When importing migrations from a package, it would be great if I could be able to flag a migration as supposing to run for tenants, rather than for the public schema.

This way, when I'm developing a package, I can Add a flag as a configuration to my package. That way when a user goes to add the latest migrations from a version update, they don't have to worry about C&P'ing them into the `tenant_migrations` folder.

**Comment by Dania02525** [comment]
Do you mean keeping them in the same /migrations folder, but respecting some @tag or something inside of the file?

The difficulty there is that Ecto parses that folder and runs all files in it against the public schema.  I think doing that would require a PR to Ecto itself. Is that what you mean?

**Comment by BenMorganIO** [comment]
@Dania02525 yep. I think letting the Ecto guys know would be helpful. I wonder if there's a way though that we could change it up from that `use Ecto.Migration` and switch it to something like `Apartmentex.Migration` and try and add the flag on Apartmentex's end.

**Comment by Dania02525** [comment]
@BenMorganIO actually, the original implementation of apartmentex looked like that `use Apartmentex.Migration` which was a light wrapper around Ecto.migration.  I would have to dig a little bit into the new code managing the migrations in Ecto to see if it could ignore modules in the same folder

**Comment by brandonparsons** [comment]
FYI - I got [this commit](https://github.com/elixir-ecto/ecto/commit/2a892ac8c15d36516c5559cfab3c77b3ed0049f2) into Ecto, which may get you somewhere on this issue.

I don't think it's in the Hex version yet.

**Comment by BenMorganIO** [comment]
Thankyou @brandonparsons. This helps :)

**Comment by BenMorganIO** [comment]
@Dania02525 here's an example of an implementation I've been working on: https://github.com/ryal/ryal/pull/3

I'm thinking I'll just add a flag to Ryal to be able to do the `mix ryal.install` into a different migration folder.
